build/scripts/docker/docker-compose-bridge.yml

```diff
@@ -1,7 +1,7 @@
 version: '3'
 services:
   bridge:
-    image: "centrifugeio/chainbridge:20201210171038-58034cb"
+    image: "centrifugeio/chainbridge:2021031218-6e5bbcf"
```

Comment:
We usually add here minutes and seconds, to know within a day the age of an image.